src/gravity/paris/HenryPeriodic.hpp

```diff
@@ -1,118 +1,39 @@
-#ifdef PARIS
```

Comment:
I don't know why Git made this a file change. This is a separate file, and PoissonPeriodic3x1DBlockedGPU.cu is deleted.